napalm/junos.py

```diff
@@ -336,16 +336,26 @@ def get_lldp_neighbors_detail(self, interface = ''):
 
         lldp_table = junos_views.junos_lldp_neighbors_detail_table(self.device)
         lldp_table.get()
-
```

Comment:
Tricky detailed view of LLDP neighbors on JunOS: https://github.com/Juniper/py-junos-eznc/issues/474